### PR TITLE
chore(deps): update dependency @testing-library/vue to v6

### PR DESCRIPTION
**Pull request by BacLuc**
This PR contains the following updates:

| Package | Type | Update | Change |
|---|---|---|---|
| [@testing-library/vue](https://togithub.com/testing-library/vue-testing-library) | devDependencies | major | [`5.8.3` -> `6.6.1`](https://renovatebot.com/diffs/npm/@testing-library%2fvue/5.8.3/6.6.1) |

---

### Release Notes

<details>
<summary>testing-library/vue-testing-library</summary>

### [`v6.6.1`](https://togithub.com/testing-library/vue-testing-library/releases/tag/v6.6.1)

[Compare Source](https://togithub.com/testing-library/vue-testing-library/compare/v6.6.0...v6.6.1)

##### Bug Fixes

-   missing optional parameter for emitted ([#&#8203;276](https://togithub.com/testing-library/vue-testing-library/issues/276)) ([6dd4847](https://togithub.com/testing-library/vue-testing-library/commit/6dd48477002760262652dd047de0897762bd60c4))

### [`v6.6.0`](https://togithub.com/testing-library/vue-testing-library/releases/tag/v6.6.0)

[Compare Source](https://togithub.com/testing-library/vue-testing-library/compare/v6.5.1...v6.6.0)

This is the Vue 3 release on `latest` channel 🎉  This means that, from now on, `npm install -D @&#8203;testing-library/vue` will install v6, which targets Vue 3.

Vue 2 is still suported with `npm install -D @&#8203;testing-library/vue@^5`.

##### Bug Fixes

-   **types:** Pass Store type to RenderOptions/render function ([#&#8203;268](https://togithub.com/testing-library/vue-testing-library/issues/268)) ([181765f](https://togithub.com/testing-library/vue-testing-library/commit/181765f9820c1927bd094c661087e5d256463a7c))
-   **types:** Remove Router and Vuex dependencies ([#&#8203;246](https://togithub.com/testing-library/vue-testing-library/issues/246)) ([9d367e7](https://togithub.com/testing-library/vue-testing-library/commit/9d367e7102f7abfce845d70349045cbb1ea6b4c0))
-   support TypeScript v3 ([#&#8203;241](https://togithub.com/testing-library/vue-testing-library/issues/241)) ([b028364](https://togithub.com/testing-library/vue-testing-library/commit/b02836439de35ae7bdc7d46c1c2783a41837bef5))
-   **types:** exclude tsconfig from types folder in publish flow ([#&#8203;213](https://togithub.com/testing-library/vue-testing-library/issues/213)) ([40caa97](https://togithub.com/testing-library/vue-testing-library/commit/40caa97ae2b7efc0da358524e5be8fb8dcdf88db))
-   Add `cleanup` type ([#&#8203;186](https://togithub.com/testing-library/vue-testing-library/issues/186)) ([75e0111](https://togithub.com/testing-library/vue-testing-library/commit/75e011176691191976d07e6b24e863268ef82446))
-   import pretty-format from testing-library/dom ([#&#8203;185](https://togithub.com/testing-library/vue-testing-library/issues/185)) ([a051013](https://togithub.com/testing-library/vue-testing-library/commit/a051013698f69493de6776cc4a06d566cc593151))
-   Make fireEvent.update work with lazy modifier ([#&#8203;183](https://togithub.com/testing-library/vue-testing-library/issues/183)) ([853d998](https://togithub.com/testing-library/vue-testing-library/commit/853d9983ce2c52f0d90999619e01313d659fc0c4))
-   **types:** allow all elements ([#&#8203;175](https://togithub.com/testing-library/vue-testing-library/issues/175)) ([a5bd6cc](https://togithub.com/testing-library/vue-testing-library/commit/a5bd6cc2453622cfc44e7abac4d179b2fd907bc6))
-   allow multiple calling of updateProps ([#&#8203;170](https://togithub.com/testing-library/vue-testing-library/issues/170)) ([95cc445](https://togithub.com/testing-library/vue-testing-library/commit/95cc44500cb14a1dca9055d5abe83d7544ab6645))

##### Features

-   Add env to turn off fireEvent.update warning ([#&#8203;181](https://togithub.com/testing-library/vue-testing-library/issues/181)) ([4376e64](https://togithub.com/testing-library/vue-testing-library/commit/4376e645dd2ecada84d5eb60c7cb8bd0ad7797d0))
-   Add support for input file on `fireEvent.update` ([#&#8203;179](https://togithub.com/testing-library/vue-testing-library/issues/179)) ([b762198](https://togithub.com/testing-library/vue-testing-library/commit/b7621983f497ce196aca50f235312ff418283fdf))
-   allow passing instantiated Vue Router ([#&#8203;239](https://togithub.com/testing-library/vue-testing-library/issues/239)) ([96c0c2d](https://togithub.com/testing-library/vue-testing-library/commit/96c0c2d8d310a4c8a9a46fd5115bf48be397b2e1))
-   allow passing instantiated Vuex store ([#&#8203;232](https://togithub.com/testing-library/vue-testing-library/issues/232)) ([9d63d71](https://togithub.com/testing-library/vue-testing-library/commit/9d63d71acf1a82565af9feacd7610501b3b6d050))
-   Move types from DefinitelyTyped into this repo ([#&#8203;165](https://togithub.com/testing-library/vue-testing-library/issues/165)) ([dbcf740](https://togithub.com/testing-library/vue-testing-library/commit/dbcf7405772b1e30b1d2c4565e46b4717528244f))
-   Warn using fireEvent.input() or .change() and suggest fireEvent.update() ([#&#8203;166](https://togithub.com/testing-library/vue-testing-library/issues/166)) ([8871d2f](https://togithub.com/testing-library/vue-testing-library/commit/8871d2f336380fa8812b834bb918313c750f6530)), closes [#&#8203;83](https://togithub.com/testing-library/vue-testing-library/issues/83)
-   **debug:** Add maxLength and options to debug ([#&#8203;164](https://togithub.com/testing-library/vue-testing-library/issues/164)) ([920dff2](https://togithub.com/testing-library/vue-testing-library/commit/920dff2c2ead320df5430812954d352c1fd0cb61))

### [`v6.5.1`](https://togithub.com/testing-library/vue-testing-library/releases/tag/v6.5.1)

[Compare Source](https://togithub.com/testing-library/vue-testing-library/compare/v6.5.0...v6.5.1)

##### Bug Fixes

-   Add missing cleanup type ([#&#8203;264](https://togithub.com/testing-library/vue-testing-library/issues/264)) ([ba14df2](https://togithub.com/testing-library/vue-testing-library/commit/ba14df2db77cdfb1f29cc972391afc86d80eca9c))

### [`v6.5.0`](https://togithub.com/testing-library/vue-testing-library/releases/tag/v6.5.0)

[Compare Source](https://togithub.com/testing-library/vue-testing-library/compare/v6.4.2...v6.5.0)

##### Features

-   drop Node 10 support ([#&#8203;243](https://togithub.com/testing-library/vue-testing-library/issues/243)) ([d704516](https://togithub.com/testing-library/vue-testing-library/commit/d704516f1b7b25e4925843ca7dcf62f26da1d940))

### [`v6.4.2`](https://togithub.com/testing-library/vue-testing-library/releases/tag/v6.4.2)

[Compare Source](https://togithub.com/testing-library/vue-testing-library/compare/v6.4.1...v6.4.2)

##### Bug Fixes

-   **types:** expose RenderOptions ([#&#8203;225](https://togithub.com/testing-library/vue-testing-library/issues/225)) ([6a27927](https://togithub.com/testing-library/vue-testing-library/commit/6a279278a04cc77e70900477a10b052e585fe045))

### [`v6.4.1`](https://togithub.com/testing-library/vue-testing-library/releases/tag/v6.4.1)

[Compare Source](https://togithub.com/testing-library/vue-testing-library/compare/v6.4.0...v6.4.1)

##### Bug Fixes

-   Fix incorrect type definition for emitted ([cbedda0](https://togithub.com/testing-library/vue-testing-library/commit/cbedda072ad27091c6c5da843e1a5e6b0f26bd5d))

### [`v6.4.0`](https://togithub.com/testing-library/vue-testing-library/releases/tag/v6.4.0)

[Compare Source](https://togithub.com/testing-library/vue-testing-library/compare/v6.3.4...v6.4.0)

##### Features

-   **Breaking Change**!: Remove Router/Vuex integration for Vue 3 ([#&#8203;206](https://togithub.com/testing-library/vue-testing-library/issues/206)) ([3c33dc3](https://togithub.com/testing-library/vue-testing-library/commit/3c33dc3dbae2098ec25fb8a0e515a67042be69df)).

More information on [https://github.com/testing-library/vue-testing-library/issues/205](https://togithub.com/testing-library/vue-testing-library/issues/205)

### [`v6.3.4`](https://togithub.com/testing-library/vue-testing-library/releases/tag/v6.3.4)

[Compare Source](https://togithub.com/testing-library/vue-testing-library/compare/v6.3.3...v6.3.4)

##### Bug Fixes

-   **install:** do not call vue-apollo patch on installation ([fa22f41](https://togithub.com/testing-library/vue-testing-library/commit/fa22f41f0882e00adf6d2af3f7c48db56137d97d))

### [`v6.3.3`](https://togithub.com/testing-library/vue-testing-library/releases/tag/v6.3.3)

[Compare Source](https://togithub.com/testing-library/vue-testing-library/compare/v6.3.2...v6.3.3)

##### Bug Fixes

-   **plugins:** allow setting vuex/router and custom plugins ([#&#8203;200](https://togithub.com/testing-library/vue-testing-library/issues/200)) ([92c3140](https://togithub.com/testing-library/vue-testing-library/commit/92c3140faec9bf38d5beb63e7d093eec6b17445c))

### [`v6.3.2`](https://togithub.com/testing-library/vue-testing-library/releases/tag/v6.3.2)

[Compare Source](https://togithub.com/testing-library/vue-testing-library/compare/v6.3.1...v6.3.2)

##### Bug Fixes

-   **rerender:** Do not mount component again on rerender ([#&#8203;199](https://togithub.com/testing-library/vue-testing-library/issues/199)) ([ca5f32c](https://togithub.com/testing-library/vue-testing-library/commit/ca5f32c22dd938d4076615e9f673bf9eeb1b18f7))

### [`v6.3.1`](https://togithub.com/testing-library/vue-testing-library/releases/tag/v6.3.1)

[Compare Source](https://togithub.com/testing-library/vue-testing-library/compare/v6.3.0...v6.3.1)

##### Bug Fixes

-   **types:** add missing types for Vue Router ([49208ef](https://togithub.com/testing-library/vue-testing-library/commit/49208ef25af03e19e5245ef592a04825e402cf29))

### [`v6.3.0`](https://togithub.com/testing-library/vue-testing-library/releases/tag/v6.3.0)

[Compare Source](https://togithub.com/testing-library/vue-testing-library/compare/v6.2.0...v6.3.0)

##### Features

-   Reenable Vue Router for Vue 3 ([#&#8203;191](https://togithub.com/testing-library/vue-testing-library/issues/191)) ([22df352](https://togithub.com/testing-library/vue-testing-library/commit/22df3526104c69bb18508bd2c0aa9df0dece4e84))

### [`v6.2.0`](https://togithub.com/testing-library/vue-testing-library/releases/tag/v6.2.0)

[Compare Source](https://togithub.com/testing-library/vue-testing-library/compare/v6.1.0...v6.2.0)

##### Features

-   Add types for Vue 3 ([#&#8203;180](https://togithub.com/testing-library/vue-testing-library/issues/180)) ([3ef7112](https://togithub.com/testing-library/vue-testing-library/commit/3ef7112c1cbf2cd7d3d1de87321aabd6dd2857dd))

### [`v6.1.0`](https://togithub.com/testing-library/vue-testing-library/releases/tag/v6.1.0)

[Compare Source](https://togithub.com/testing-library/vue-testing-library/compare/v6.0.0...v6.1.0)

##### Features (breaking)

-   Replace `setProps` with `rerender` ([#&#8203;173](https://togithub.com/testing-library/vue-testing-library/issues/173)) ([ebe23d8](https://togithub.com/testing-library/vue-testing-library/commit/ebe23d855ce95b6a5019704e19bd5745428120d8))

### [`v6.0.0`](https://togithub.com/testing-library/vue-testing-library/releases/tag/v6.0.0)

[Compare Source](https://togithub.com/testing-library/vue-testing-library/compare/v5.8.3...v6.0.0)

**tl;dr**: Run `npm install -D @&#8203;testing-library/vue@next` to give this release which supports Vue 3 a go!

##### Features

-   Update to support Vue 3.0 and Vue Test Utils 2.0 ([#&#8203;137](https://togithub.com/testing-library/vue-testing-library/issues/137)) ([d9b1b5e](https://togithub.com/testing-library/vue-testing-library/commit/d9b1b5e0c049d07e21b65e00668dbde7349e26c5))

##### BREAKING CHANGES

-   Snapshots are potentially different due to whitespaces.
-   Some options such as stubs are now moved to `global.stubs`.
-   Previous usage of vue instance as a parameter for the callback function is now replaced with global `config` options (for example, `global.directives`).
-   `isUnmounted` is gone.
-   `updateProps` is now called `setProps` to align with VTU.

Some of the missing steps is to provide support to Vue Router, and also to update libraries of the ecosystem to their Vue 3 version.

This is an exciting release! 🎉  It marks the first release aiming to support Vue 3 and Vue Test Utils 2. Please [head to the PR](https://togithub.com/testing-library/vue-testing-library/pull/137) to get more information, and feel free to open up issues and PRs to fix missing features / ports 😄

</details>

---

### Configuration

📅 **Schedule**: Branch creation - At any time (no schedule defined), Automerge - At any time (no schedule defined).

🚦 **Automerge**: Disabled by config. Please merge this manually once you are satisfied.

♻ **Rebasing**: Whenever PR becomes conflicted, or you tick the rebase/retry checkbox.

🔕 **Ignore**: Close this PR and you won't be reminded about this update again.

---

 - [ ] If you want to rebase/retry this PR, click this checkbox.

---

This PR has been generated by [Renovate Bot](https://togithub.com/renovatebot/renovate).

### CODE REVIEW

**Comment by BacLuc** [comment]
### Renovate Ignore Notification

As this PR has been closed unmerged, Renovate will ignore this upgrade and you will not receive PRs for *any* future 6.x releases. However, if you upgrade to 6.x manually then Renovate will reenable minor and patch updates automatically.

If this PR was closed by mistake or you changed your mind, you can simply rename this PR and you will soon get a fresh replacement PR opened.
